src/app/keyboard/components/virtual-keyboard/virtual-keyboard.component.html

```diff
@@ -0,0 +1,13 @@
+<div
+    *ngFor="let keyBinding of keyBindings" (click)="keyBinding.onMatched()"
+    class="key"
+    [class.key--matched]="isKeyBindingMatched(keyBinding)"
+>
+    <span class="key-label">{{ keyBinding.label }}</span>
+    <span class="key-keys">{{ displayKeyBinding(keyBinding) }}</span>
+</div>
+
+<pre class="keystrokes">
+Pressed:keys
+{{ keystrokes }}
+</pre>
```

Comment:
Is this still supposed to be here?

Reply:
Yes. I will remove it when the virtual keyboard is implemented :)